### ISSUE TITLE
Installation just crashes

### ISSUE THREAD

**Issue by toluschr** [pastapojken/Fusion360WineInstaller#1]
So I really want this to work, but after typing `./installer.sh 1` into a terminal the installation window appears, but the process crashes. I never get to the point where I can login to my account. I tried continuing the steps anyways, and the launcher does show up anyways but it crashes too. (Wine 4.6-staging)

**Comment by pastapojken** [comment]
Fusion is a bit hit or miss, I myself have been doing testing as, since ```corefonts winhttp wininet``` were added, installation has been broken for me too (which is funny because I re-added those as they were needed for someone else's install to work).  I've just updated the readme with more instructions (note: not sure any of those work, haven't had time to do proper testing), if those don't help, i suggest editing the installer script and removing ```corefonts winhttp wininet```, as that might help. 
Also what distro are you on?
